package.json

```diff
@@ -45,9 +47,12 @@
 		"typescript": "^4.6.3"
 	},
 	"dependencies": {
-		"ardrive-core-js": "^1.12.0",
-		"arweave": "1.10.18",
-		"node-fetch": "^2.6.1"
+		"arweave": "1.10.23",
+		"axios": "^0.26.1",
+		"node-fetch": "^2.6.1",
```

Comment:
Consider using axios exclusively. We dropped node-fetch from core recently. There not much need for two different network dependencies

Reply:
Nice point

Reply:
49e06418145160af8fc037cc77316ea282163b13 and 4378a52e1f7aa55311081fb04d27171661596843